CMakeLists.txt

```diff
@@ -84,9 +92,25 @@ if(ENABLE_TESTS)
    set(CATCH_BUILD_TESTING OFF CACHE BOOL "disable building tests")
    set(CATCH_INSTALL_DOCS OFF CACHE BOOL "disable installing docs")
    set(CATCH_INSTALL_HELPERS OFF CACHE BOOL "disable installing helpers")
-   add_subdirectory(${CMAKE_SOURCE_DIR}/external/Catch2)
+   add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/external/Catch2)
    include(external/Catch2/contrib/Catch.cmake)
    add_subdirectory(tests)
+   if (NOT EXISTS ${CMAKE_BINARY_DIR}/wasms)
```

Comment:
Note: This allows us to avoid having to clone on Travis. Travis doesn't have any git keys/logins so it will fail to clone and therefore fail the build. Once wasms is a public repo we can remove it